app/src/main/java/com/github/houseorganizer/houseorganizer/MainScreenActivity.java

```diff
@@ -0,0 +1,34 @@
+package com.github.houseorganizer.houseorganizer;
+
+import android.os.Bundle;
+import android.view.View;
+import android.widget.TextView;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+public class MainScreenActivity extends AppCompatActivity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_main_screen);
+    }
+
+    public void houseButtonPressed(View view){
+        TextView text = findViewById(R.id.last_button_activated);
+        String s = "House button pressed";
```

Comment:
(Question) It might be better to store this string as a resource string in res/values/strings.xml (same for the other two strings), or maybe it's better if we keep them here since they won't be part of the end product (easier to delete) ?

Reply:
As this is only here for testing purposes and probably be gone next week, I think we can keep them here so we don't have to modify multiple files when getting rid of it.